Source/WebKit/UIProcess/AuxiliaryProcessProxy.h

```diff
@@ -153,6 +153,9 @@ class AuxiliaryProcessProxy : public ThreadSafeRefCounted<AuxiliaryProcessProxy,
     void ref() final { ThreadSafeRefCounted::ref(); }
     void deref() final { ThreadSafeRefCounted::deref(); }
 
+    bool operator==(const AuxiliaryProcessProxy& other) const { return (this == &other); }
```

Comment:
Was this needed because of any of your changes?

Reply:
Yeah, I loop through the processes and check equality. This makes that cleaner.